lib/rubocop/cop/layout/block_end_newline.rb

```diff
@@ -55,6 +56,18 @@ def indentation_of_block_start_line(node)
           match = /\S.*/.match(node.loc.begin.source_line)
           match.begin(0)
         end
+
+        def range_to_replace(node)
```

Comment:
Could you rename this to something that isn't coupled to the context of its use in `#autocorrect`? 🙂

Reply:
Good point. How about `delimiter_range`?